src/Model/ModelManager.php

```diff
@@ -634,10 +634,12 @@ protected function camelize($property)
      */
     private function getValueFromType($value, Type $type, string $fieldType, AbstractPlatform $platform): string
     {
-        if ('binary' === $platform->getDoctrineTypeMapping($fieldType)) {
+        if ($platform->hasDoctrineTypeMappingFor($fieldType) &&
+            'binary' === $platform->getDoctrineTypeMapping($fieldType)
+        ) {
             return (string) $type->convertToPHPValue($value, $platform);
         }
 
-        return $type->convertToDatabaseValue($value, $platform);
+        return (string) $type->convertToDatabaseValue($value, $platform);
```

Comment:
Why is the casting required?
Could you please add a test covering this fix?

Reply:
The return type of [`Doctrine\DBAL\Types\Type::convertToDatabaseValue()`](https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Types/Type.php#L144) may be a mixed and this violates the declared return type of `Sonata\DoctrineORMAdminBundle\Model\ModelManager::getValueFromType()`. So, we must either cast the value to a string or change the return type declaration.

Reply:
@phansys i update tests